index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="fr">
+
+<head>
+    <!-- En-tête de la page-->
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ohmyfood</title>
+    <meta name="description" content="Découvrez des menus qui vous conviennent dans des restaurants d'execption">
+    <meta name="keyword" content="gastronomie, menu , restaurant, gastronomique, execption">
+    <!--Police-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,700&family=Shrikhand&display=swap"
+        rel="stylesheet">
+    <!--Style-->
+    <link rel="stylesheet" href="./public/css/style.css">
+    <!--FontAwesome-->
+    <script src="https://kit.fontawesome.com/283482e73e.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <!--header-->
+    <header>
+        <a href="index.html"><img src="./images/logo/ohmyfood@2x.svg" alt="Logo Ohmyfood"></a>
+    </header>
+    <main>
+        <!--search-bar-->
+        <section id="search-bar">
+            <span><i class="fas fa-map-marker-alt"></i></span>
+            <input type="search" name="city" value="Paris,Belleville" placeholder="Votre ville">
+            <h2> Réservez le menu qui vous convient </h2>
+            <p> Découvrez des restaurants d'exception, sélectionnés par nos soins </p>
+            <button> Explorer nos restaurants </button>
+        </section>
+        <!--menu-->
+        <section id="menu">
+            <h2>Fonctionnement</h2>
+            <article>
+                <ul>
+                    <li>1</li>
+                    <li><i class="fas fa-mobile-alt operation__steps--icon"></i></li>
+                    <li>Choisissez un restaurant</li>
+                </ul>
+            </article>
+            <article>
+                <ul>
+                    <li>2</li>
+                    <li><i class="fas fa-list-ul operation__steps--icon"></i></li>
+                    </lio>
+                    <li>Composez votre menu</li>
+                </ul>
+            </article>
+            <article>
+                <ul>
+                    <li>3</li>
+                    <li><i class="fas fa-store operation__steps--icon"></i></li>
+                    <li>Dégustez au restaurant</li>
+                </ul>
+            </article>
```

Comment:
No article needed juste an ol with li and not li because here we use numbers. Eache li must only have icon and text, we'll add number later with css